app/components/TumourSummaryEdit/index.tsx

```diff
@@ -323,32 +339,27 @@ const TumourSummaryEdit = ({
     return null;
   }, [handleClicked, handleDelete, handleKeyDown, newMicrobialData]);
 
-  const mutBurDataSection = useMemo(() => {
-    if (newMutationBurdenData) {
-      return (
-        <TextField
-          className="tumour-dialog__text-field"
-          label="Mutation Burden (Mut/Mb)"
-          value={newMutationBurdenData.totalMutationsPerMb}
-          name="totalMutationsPerMb"
-          onChange={handleMutationBurdenChange}
-          variant="outlined"
-          fullWidth
-          type="number"
-        />
-      );
-    }
-    return null;
-  }, [newMutationBurdenData, handleMutationBurdenChange]);
+  const mutBurDataSection = (
```

Comment:
Hi Jacky, is there a reason why use don't use the useMemo() hook anymore for this section specifically? Thanks!

Reply:
That's a good question actually, I'm not too sure about the numbers but I removed useMemo here because the rendering of this one particular component doesn't seem worth enough (in terms of recalculation efficiency) to put into a memo hook